### PR TITLE
first two section is which is menu and also headline section styling is done

### PR DESCRIPTION
**Pull request by MMhaidari**
# what is added new
-  menu section
-  headline section
-  main header 
-  headline description
-  social media icons

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @MMhaidari, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid

You have submitted a project with failing linter checks. Kindly fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MMhaidari** [comment]
# what is added new
- menu section
- headline section
- main header
- headline description
- social media icons
- linter errors solved

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <title>Profilo Website</title>
+  </head>
+  <body>
+    <div class="container">
+      <section class="section-header">
+        <nav class="navigation">
+          <img src="mobile-ingrediants/My Logo.png" alt="logo" />
+          <img src="mobile-ingrediants/Union.png" alt="menu-btn" />
+        </nav>
```
Comment (Kamba56) [inline index.html:22]: - [ ] Please use the `header` element to wrap the header section of your document. Kindly note that the header section is separate from the headline section, so you can use a `section` element to wrap the headline section.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      rel="stylesheet" />
+
+    <link rel="stylesheet" href="style.css" />
+    <title>Profilo Website</title>
+  </head>
+  <body>
+    <div class="container">
+      <section class="section-header">
+        <nav class="navigation">
+          <img src="mobile-ingrediants/My Logo.png" alt="logo" />
```
Comment (Kamba56) [inline index.html:20]: - [ ] Kindly consider wrapping the logo in an `a` tag with `href` attribute pointing to index.html. This is semantically correct and also improves your UX,

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          <img src="mobile-ingrediants/Union.png" alt="menu-btn" />
```
Comment (Kamba56) [inline index.html:21]: - [ ] Kindly consider wrapping the hamburger button in a `burton` element or an `a` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          alt="headline-img" /> -->
+          <h1 class="primary-header">I'm Mohsen Glad to see you!</h1>
+          <p class="primary-decrption">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h4 class="secondary-header">LET’S CONNECT</h4>
+          <div class="links">
```
Comment (Kamba56) [inline index.html:35]: - [ ] Please make sure the list of icons is an unordered list as demanded by the project requirements. It should be a ul containing li which has the a tag inside.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @MMhaidari , 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MMhaidari** [comment]
# what is added new
- header section wrapped up with header tag
- logo wrapped up with a tag 
- Hemberger menu button wrapped up with btn tag 
- social media links wrapped up with ul list tag

**Review by frank1738 — APPROVED** [review]
Hi @MMhaidari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Optional suggestions

 - [ ] You can consider positioning your background images correctly so that they don't appear to be colliding with the other elements of your page
 
image




_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**